src/client/decorators/MetaDecorator/DiagramDesigner/AttributeDetailsDialog.js

```diff
@@ -8,8 +8,9 @@ define([
     'js/util',
     'common/regexp',
     'text!./templates/AttributeDetailsDialog.html',
+    'js/Dialogs/CodeEditor/constants',
     'css!./styles/AttributeDetailsDialog.css'
-], function (util, REGEXP, attributeDetailsDialogTemplate) {
+], function (util, REGEXP, attributeDetailsDialogTemplate, MODE_CONSTANTS) {
```

Comment:
The MetaDecorator should use a enumeration from `common/Constants.js`, the js/Dialogs/CodeEditor/constants should provide a mapping from that to the code-editor language.